internal/provider/resource_managed_identity_alias.go

```diff
@@ -0,0 +1,281 @@
+	ttypes "gitlab.com/infor-cloud/martian-cloud/tharsis/tharsis-sdk-go/pkg/types"
+)
+
+// ManagedIdentityAliasModel is the model for a managed identity alias.
+type ManagedIdentityAliasModel struct {
+	ID            types.String `tfsdk:"id"`
+	ResourcePath  types.String `tfsdk:"resource_path"`
+	Name          types.String `tfsdk:"name"`
+	GroupPath     types.String `tfsdk:"group_path"`
+	LastUpdated   types.String `tfsdk:"last_updated"`
+	AliasSourceID types.String `tfsdk:"alias_source_id"`
+}
+
+// Ensure provider defined types fully satisfy framework interfaces
+var (
+	_ resource.Resource                = (*managedIdentityAliasResource)(nil)
+	_ resource.ResourceWithConfigure   = (*managedIdentityAliasResource)(nil)
+	_ resource.ResourceWithImportState = (*managedIdentityAliasResource)(nil)
+)
+
+// NewManagedIdentityAliasResource is a helper function to simplify the provider implementation.
+func NewManagedIdentityAliasResource() resource.Resource {
+	return &managedIdentityAliasResource{}
+}
+
+type managedIdentityAliasResource struct {
+	client *tharsis.Client
+}
+
+// Metadata returns the full name of the resource, including prefix, underscore, instance name.
+func (t *managedIdentityAliasResource) Metadata(ctx context.Context,
+	req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = "tharsis_managed_identity_alias"
+}
+
+// The diagnostics return value is required by the interface even though this function returns only nil.
+func (t *managedIdentityAliasResource) GetSchema(ctx context.Context) (tfsdk.Schema, diag.Diagnostics) {
+	description := "Defines and manages a managed identity alias."
+
+	return tfsdk.Schema{
+		Version: 1,
+
+		MarkdownDescription: description,
+		Description:         description,
+
+		Attributes: map[string]tfsdk.Attribute{
+			"id": {
+				Type:                types.StringType,
+				MarkdownDescription: "String identifier of the managed identity alias.",
+				Description:         "String identifier of the managed identity alias.",
+				Computed:            true,
+				PlanModifiers: tfsdk.AttributePlanModifiers{
+					resource.UseStateForUnknown(),
+				},
+			},
+			"resource_path": {
+				Type:                types.StringType,
+				MarkdownDescription: "The path of the parent group plus the name of the managed identity alias.",
+				Description:         "The path of the parent group plus the name of the managed identity alias.",
+				Computed:            true,
+				PlanModifiers: tfsdk.AttributePlanModifiers{
+					resource.UseStateForUnknown(),
+				},
+			},
+			"name": {
+				Type:                types.StringType,
+				MarkdownDescription: "The name of the managed identity alias.",
+				Description:         "The name of the managed identity alias.",
+				Required:            true,
+			},
+			"group_path": {
+				Type:                types.StringType,
+				MarkdownDescription: "Full path of the parent group.",
+				Description:         "Full path of the parent group.",
```

Comment:
This description can specify that this is the group path for where the alias will be created.